pkgs/servers/code-server/default.nix

```diff
@@ -55,18 +55,19 @@ let
     ln -s -f ${esbuild'}/bin/esbuild ${path}/node_modules/esbuild/bin/esbuild
   '';
 
-  commit = "2798322b03e7f446f59c5142215c11711ed7a427";
+  # Todo: find a way to get this commit from `src` ?
+  commit = "70aa1b77226ea40e5d661103de7b354f742a76df";
```

Comment:
@code-asher Can you give me some indication on how to get this commit ?

I took the one corresponding to the tag, but is it the proper way ?

Reply:
Yup pulling it from the tag is the proper way.  It should be `5c199629305a0b935b4388b7db549f77eae82b5a` I believe?

```console
λ git rev-parse v4.14.1
5c199629305a0b935b4388b7db549f77eae82b5a
```

It would be nice if we could just run `git rev-parse` as part of the build but I assume that will not work here since whoever came up with the original expression explicitly patched that out.

Reply:
Do you think we can remove that feature with a patch? What's the point of having that commit?

Reply:
I updated the patch, it works without the need of that commit now

Reply:
If the commit is missing it will break display languages (Japanese, Spanish, etc).  For some reason VS Code has a hard dependency on the commit being set for that functionality, although I am not sure why.  :shrug:  

The second reason is that the commit is used in cache busting.  Without the commit you could run into issues where the browser is loading old versions of assets from the cache.

The third reason is that it can be helpful for the commit to be accurate in bug reports especially when they are built outside of our CI as sometimes the version numbers can be unreliable (since they are arbitrarily provided).  For example, the Termux package (not maintained by us) currently shows `0.0.0` for the version for some reason but since we have the commit we still know what the version actually is when bugs are submitted.  It can also be helpful if the bug report comes from a development build although that does not apply here.

Reply:
Right. I will create a PR fixing this immediately. Thanks for letting me know.

Reply:
Created a new PR, feel free to approve it or request changes at https://github.com/NixOS/nixpkgs/pull/240194